tests/nimble_autoconn_gnrc/Makefile

```diff
@@ -7,7 +7,7 @@ USEMODULE += shell_commands
 USEMODULE += ps
 
 # Include GNRC and RPL
-USEMODULE += gnrc_netdev_default
+USEMODULE += netdev_default
```

Comment:
Same here. Not a GNRC module anymore + I am not sure it is needed or should be replaced by `nimble` directly.

Reply:
`nimble_autoconn_ipsp` already pulls in `nimble`